tests/framework/installer/installer.go

```diff
@@ -85,10 +87,19 @@ func init() {
 		createBaseTestDir = false
 		baseTestDir = "/data"
 	}
+	if RookTestOpenshift {
+		baseTestDir = "/var/lib/rook"
```

Comment:
Why does this need to be `/var/lib/rook` on openshift? Can we make this an environment variable instead of hard coding for openshift?

Reply:
@travisn this I saw is set here: 
https://github.com/rook/rook/blob/82702f5cc5d6150fbcc3990045a75379338b1d96/cluster/examples/kubernetes/ceph/cluster.yaml#L28-L31
like this as well and as I remember correctly I was failing on openshift when it was set to /data . So I am using what is set also in mentioned example as I see /data is mentioned for minikube.

Reply:
@petr-balogh What about a new env variable? The test directory could be different for various OpenShift or Kubernetes environments, so it doesn't seem like we should tie it to the RookTestOpenshift variable.  
```
  if RookBaseTestDir != "" {
     baseTestDir = RookBaseTestDir
     fmt.Sprintf("baseTestDir set to %s", baseTestDir)
  } 
```